spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/repository/query/PartTreeFirestoreQuery.java

```diff
@@ -64,13 +66,18 @@
 
 	private final FirestoreClassMapper classMapper;
 
-	private static final Map<Part.Type, StructuredQuery.FieldFilter.Operator> PART_TO_FILTER_OP =
-			new MapBuilder<Part.Type, StructuredQuery.FieldFilter.Operator>()
-					.put(SIMPLE_PROPERTY, StructuredQuery.FieldFilter.Operator.EQUAL)
-					.put(GREATER_THAN_EQUAL, StructuredQuery.FieldFilter.Operator.GREATER_THAN_OR_EQUAL)
-					.put(GREATER_THAN, StructuredQuery.FieldFilter.Operator.GREATER_THAN)
-					.put(LESS_THAN_EQUAL, StructuredQuery.FieldFilter.Operator.LESS_THAN_OR_EQUAL)
-					.put(LESS_THAN, StructuredQuery.FieldFilter.Operator.LESS_THAN)
+	private static final Map<Part.Type, OperatorSelector> PART_TO_FILTER_OP =
```

Comment:
I think this should move to the `OperatorSelector` class for better encapsulation.